### PR TITLE
Remove intermediate `_clustered`/`_consensus` directories/files from pixel and cell clustering pipeline

### LINKED ISSUE #571: Remove intermediate `pixel_mat_clustered`/`cell_mat_clustered` directory/file
**Describe the solution you'd like**

`pixel_mat_clustered` contains the same data as `pixel_mat_consensus`, just without the meta cluster columns. Now that we're working with large datasets, we should not need to keep large duplicate datasets around.

We write to one `pixel_mat_consensus` directory with just the SOM labels at first. It will be much more intuitive to overwrite the old `.feather` files with the pixel data containing the meta cluster labels once that step is performed. This removes an additional variable from the notebook and also simplifies the directory structure prior to finalizing the pipeline for pixel and cell clustering.

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Closes #571. There is no need to create a separate `pixel_mat_clustered` directory/`cell_mat_clustered` file as the data is just the channel expression/pixel cluster count expression values with an additional `pixel_som_cluster`/`cell_som_cluster` attached. Ditto for the `pixel_mat_consensus` directory/`cell_mat_consensus` file. These should be removed from the pixel and cell clustering processes.

**How did you implement your changes**

We should overwrite the original data with the newly created data in the `cluster_pixels`/`pixel_consensus_cluster` (and similarly for the cell clustering functions).

**Remaining issues**

There are a few extra checks we need to include for validation if the user decides to rerun the clustering pipeline in the same session.

### CODE REVIEW

**Comment by alex-l-kong** [comment]
This branch is almost done, need to verify the outputs match up compared to the original.

**Review by ngreenwald — COMMENTED** [review]
This looks great. Did you already run both notebooks on the HIV dataset to check that everything is working?

**Comment by alex-l-kong** [comment]
@ngreenwald yes, I'm currently running one more test to be sure.

File: templates_ark/example_cell_clustering.ipynb [hunk 1]
```diff
@@ -77,10 +77,12 @@
     "base_dir = \"../data/granulomaCohort_allData\"\n",
```
Comment (ngreenwald) [inline templates_ark/example_cell_clustering.ipynb:77]: <pre class="ql-syntax" spellcheck="false">Line #6.    pixel_output_dir = 'test_candace_condense_pixel_output_dir'
</pre><p>This needs to be changed back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, just one leftover path change in the notebook

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks like some merge conflicts from the normalization PR I just put in

**Comment by alex-l-kong** [comment]
@ngreenwald resolved

**Review by ngreenwald — APPROVED** [review]

**Pull request merged**